notebooks/lesson06_model-interpretation.ipynb

```diff
@@ -0,0 +1,2088 @@
+{
```

Comment:
<p>a cool exercie could be to investigate that data entry with 1200 people.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
<p>i agree. will add it as an exercise inside the solution :D</p>